_studio/mfx_lib/scheduler/src/mfx_scheduler_core_task_management.cpp

```diff
@@ -296,7 +296,7 @@ bool mfxSchedulerCore::IsReadyToRun(MFX_SCHEDULER_TASK *pTask)
         // let the thread inspect other tasks.
         if (pTask->param.occupancy) {
             return false;
-        } else {
+        } else if (m_timeWaitPeriod) {
```

Comment:
I'm anticipating questions about this if() . The intention is to disable code block below and at the same time to simplify merge with Windows in the future.

Reply:
Without this you will keep trying to execute the task which returned MFX_TASK_BUSY again and again. I.e. there is a risk of polling. We need to evaluate whether such polling can or can not happen.